### PR TITLE
Categories

### LINKED ISSUE #44: Group services depending on function, purpose or nature
As of today there are 21 services visible on findit.chalmers.it which fills my entire screen and some more. To more easily navigate among these cards/link I think it would be favourable to group them with similar cards. A proposal can be seen below.

Hubben related services:
 * Bookit
 * hubbot
 * hueIT
 * singIT
 * VulcanTV
 * SuggestIT

Other Websites:
 * BussIT
 * Chalmers.it
 * eatIT
 * HowTo
 * LinkIT
 * mat
 * sannes
 * songbook

Various Links:
 * findIT
 * GDPR
 * Github
 * IRC tutorial
 * IT event calendar
 * slack
 * wikITpedia

### PR DESCRIPTION
**Pull request by antoneekstrom**
resolve #44
I added a category property to each service and sort them under their respective label. Also added a sort of sidebar with links to each category. I slightly changed the suggested categories by adding the following: Food, Tools and Information. Though, I am sure we could come up with some more fitting categories.

### CODE REVIEW

File: frontend/src/components/elements/service/styles.jsx [hunk 1]
```diff
@@ -4,4 +4,4 @@ export const ServiceIcon = styled.i`
     color: #777777;
     font-size: 28px;
     margin-right: 7px;
-`;
+`;
```
Comment (Hoidi) [inline frontend/src/components/elements/service/styles.jsx:7]: New line is nice

File: frontend/src/components/elements/service/index.jsx [hunk 2]
```diff
@@ -43,7 +43,7 @@ export const Service = ({
                     target={"_blank"}
                     rel="noopener noreferrer"
                 >
-                    <DigitText.Heading5 text={title} bold />
+                    <DigitText.Heading5 text={title} bold="true" />
```
Comment (Hoidi) [inline frontend/src/components/elements/service/index.jsx:46]: I'm pretty sure this is the same thing

File: frontend/src/service.json [hunk 3]
```diff
@@ -4,127 +4,143 @@
...
     
     {
         "title": "IRC tutorial",
         "url": "https://irc.chalmers.it",
         "description": "Mega awesome digIT IRC-tutorial.",
         "github_url": "",
-        "fontawesome_icon": "fa-comments-o"
+        "fontawesome_icon": "fa-comments-o",
+        "category": "Information"
     },
```
Comment (Hoidi) [inline frontend/src/service.json]: digIT should probably go over some of these and fix/remove them

File: frontend/src/service.json [hunk 4]
```diff
@@ -133,44 +149,50 @@
...
         "description": "The official digital songbook of the Software Engineering Student Division (IT).",
         "github_url": "https://github.com/cthit/songbook",
-        "fontawesome_icon": "fa-music"
+        "fontawesome_icon": "fa-music",
+        "category": "Hubben"
     },
     {
         "title": "hexIT",
         "url": "https://hex.chalmers.it/",
         "description": "The official color palette for IT",
```
Comment (Hoidi) [inline frontend/src/service.json]: I know this isn't changed in this PR but 09cdda is not an official hexcolor for the IT section. There are none actually

**Review by Hoidi — CHANGES_REQUESTED** [review]
Good job! Looks pretty hot but I dislike the implementation of the navigation. It doesn't work well with phones and it looks kinda slapped on. Maybe put it in a hamburger menu top left? Maybe a [DigitHeaderDrawer](https://cthit.github.io/react-digit-components/?path=/docs/digitheaderdrawer--simple-header)

The categories also seems a little wack. But not too insane

File: frontend/src/components/elements/service/index.jsx (hunk 2 above)
Comment (lindskogen) [reply frontend/src/components/elements/service/index.jsx:46]: But.. a string?

Comment (antoneekstrom) [reply frontend/src/components/elements/service/index.jsx:46]: Yeah I don't know, I noticed this error message in the console so I thought I'd change it.
```
Warning: Received 'true' for a non-boolean attribute 'bold'
If you want to write it to the DOM, pass a string instead: bold="true" or bold={value.toString()}.
```

**Comment by antoneekstrom** [comment]
@Hoidi A `DigitHeaderDrawer` would be better, I did see it when looking through the storybook so I'm not sure why I didn't opt for that instead.

**Comment by antoneekstrom** [comment]
I changed to the drawer instead. It is definitely better, but it does still feel a little weird, may just be me though.
It works better on mobile with the drawer. I would like to change the `Grid` to a `Row` on mobile for the cards. I was wondering if there are any guidelines on mediaqueries, if there is a method we usually use?

**Comment by Hoidi** [comment]
> I was wondering if there are any guidelines on mediaqueries, if there is a method we usually use?

Don't think so. But I have no idea

**Review by Hoidi — APPROVED** [review]

**Review by simonengstrm — APPROVED** [review]
I approve these changes.

I like it and it looks good 😎

**Comment by Hoidi** [comment]
Merge @cthit/digit ? @zimbosaurus @LilleLink

**Comment by antoneekstrom** [comment]
@Hoidi Yes!

**Pull request merged**